lighthouse-core/lib/url-shim.js

```diff
@@ -23,7 +23,7 @@
 
 /* global self */
 
-const ELLIPSIS = '\u2026';
+const Util = require('../report/v2/renderer/util.js');
```

Comment:
Actually, looking at the tree, util.js is already that file. We could move it out of /renderer if location is a big deal, but then we'll need to add it to closure checking.